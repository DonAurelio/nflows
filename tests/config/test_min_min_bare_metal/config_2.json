{
    "dag_file": "./tests/workflows/test_min_min_bare_metal/config_2.dot",
    "scheduler_type": "min_min",
    "mapper_type": "bare_metal",
    "mapper_mem_policy_type": "default",

    "core_avail_mask": "0x01000001",
    "flops_per_cycle": 32,
    "clock_frequency_type": "static",
    "clock_frequency_hz": 1000000000,

    "distance_matrices": {
        "latency": "./tests/system/test_min_min_bare_metal/2_lat.txt",
        "bandwidth": "./tests/system/test_min_min_bare_metal/2_bw.txt"
    },

    "log_base_name": "./tests/output/test_min_min_bare_metal/config_2",
    "log_date_format": ""
}
