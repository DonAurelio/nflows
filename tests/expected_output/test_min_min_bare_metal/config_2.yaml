core_availability:
  0: {avail_until: 0}
  24: {avail_until: 0}

name_to_thread_locality:
  Task_3: {numa_id: 1, core_id: 24}
  Task_2: {numa_id: 1, core_id: 24}
  Task_1: {numa_id: 0, core_id: 0}
